### ISSUE TITLE
[BUG]: <figure> element must have an associated label

### ISSUE THREAD

**Issue by jamesjacobs** [IBMa/equal-access#1331]
### Project

a11y checker extension

### Browser

Chrome

### Operating system

MacOS

### Description

The Equal Access Accessibility Checker Chrome extension is reporting that `<figure>` elements must have an accessible label when a `<figure>` either contains an empty `<figcaption>` or does not contain a `<figcaption>`.

`<figcaption>`'s are optional and not a requirement: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure.

![image](https://user-images.githubusercontent.com/756393/222121768-acb44fbc-26e6-4520-adb7-b68b879d0160.png)

### Steps to reproduce

Set ruleset to WCAG 2.1 (A, AA).

Example mark up to trigger failure:

```
<figure>
    <img src="https://picsum.photos/200/300" alt="some alt text">
    <figcaption></figcaption>
</figure>

<figure>
    <img src="https://picsum.photos/200/300" alt="some alt text>
</figure>
```

**Comment by tombrunet** [comment]
We'll take a look. Might be a candidate to drop to recommendation. That said, the alt of the image is not the accessible name of the figure. It really doesn't make sense to use figure at all in that scenario since without a caption, it's not a figure, it's just an image. 
@scottaohara has a good writeup of it here: https://www.scottohara.me/blog/2019/01/21/how-do-you-figure.html

**Comment by aliunwala** [comment]
Triage:
We should downgrade this from violation to recommendation.  Captions are not required for figures (Seems strange... but the spec doesn't enforce this. So we are incorrectly flagging this as a violation)

**Comment by sct-will** [comment]
Gated by build issue.

**Comment by ErickRenteria** [comment]
Merged
